lldb/tools/lldb-dap/lldb-dap.cpp

```diff
@@ -1710,6 +1710,8 @@ void request_initialize(const llvm::json::Object &request) {
   body.try_emplace("supportsLogPoints", true);
   // The debug adapter supports data watchpoints.
   body.try_emplace("supportsDataBreakpoints", true);
+  // Putting version string. Note: this is not part of DAP spec.
+  body.try_emplace("version", g_dap.debugger.GetVersionString());
```

Comment:
I would add a dictionary named "__lldb" and the contents will be the "version": <string> like:
```
"__lldb": {
  "version": "<version string>"
}
```
Then we can add more stuff in here in the future if we need to  know things. The Python version would be interesting as well possibly in the future.